lib/winston-syslog.js

```diff
@@ -342,9 +358,17 @@ class Syslog extends Transport {
     if (this.isDgram) {
       return this.connectDgram(callback);
     }
+    // Set Secure options for TLS
+    const options = {
+      servername: this.host,
+      checkServerIdentity: () => this.checkServerIdentity, // check FQDN
+      secureProtocol: this.secureProtocol,
```

Comment:
This is described in the documentation as a legacy mechanism, and may be deprecated in the future.  Might it be better to not provide a default for this value above, but still let it be passed through if provided?

Reply:
I don't think so. They have not mentioned yet that it will be deprecated in the future. 

Reply:
Anyway, we can remove the default value and get values from users.

Reply:
I remove servername: this.host, checkServerIdentity: () => this.checkServerIdentity, // check FQDN.  I think they need to be investigated again. I will add them in the future if they are needed. 